app-api/src/main/resources/liquibase-sender.xml

```diff
@@ -75,6 +75,41 @@
         </createTable>
     </changeSet>
 
+ <changeSet author="jpboane" id="20210511-1200">
```

Comment:
Why is this here? This PR looks incorrect, can you please create a new branch off the latest master and make the Portuguese changes in the new branch? You can copy your translated file from this branch to the new branch